test/functional/test_framework/test_framework.py

```diff
@@ -451,7 +457,7 @@ def _initialize_chain(self, extra_args=None, stderr=None):
                         self.nodes[peer].generate(1)
                         block_time += 156
                     # Must sync before next peer starts generating blocks
-                    sync_blocks(self.nodes)
+                    self.sync_blocks(self.nodes)
```

Comment:
noting difference https://github.com/bitcoin/bitcoin/pull/15773/commits/fa6dc7c5c3477bda04481f080d430f63539974e2#diff-64721c5ee64d44f7114d6d0d2226db4dR503

```suggestion
                    self.sync_blocks()
```

Reply:
Fixed/squashed